spec/queue_test.js

```diff
@@ -0,0 +1,31 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Stack from '../src/queue'
+
+chai.use(chaiChange)
+
+describe('Queue', () => {
+'use strict'
+
+// it('exists', () => {
```

Comment:
Before you submit your work don't leave comments unless it is something specific to give note for the reviewer.